presto-docs/src/main/sphinx/release/release-0.247.rst

```diff
@@ -0,0 +1,39 @@
+=============
+Release 0.247
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug for reporting output data sizes for optimized repartitioning.
+* Fix accounting for revocable memory that could cause some queries not to spill when they should.
+* Fixed a race condition in enum key lookup which caused queries using the `ENUM_KEY` UDF to crash occasionally with an internal error #15607.
+* Add function ``array_intersect`` that takes an array of array as input.
```

Comment:
we mark function using the syntax in https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#formatting

so 

```
:func:`array_intersect`
```

Also, lets change to `Add an implementation of ` since this function already exists.
Also `array of arrays`